lib/spack/spack/cmd/checksum.py

```diff
@@ -28,12 +28,20 @@ def setup_parser(subparser):
     subparser.add_argument(
         '--keep-stage', action='store_true',
         help="don't clean up staging area when command completes")
+    subparser.add_argument(
+        '-y', '--yes', action='store_true',
+        help='non-interactive mode')
```

Comment:
Make sure you update `share/spack/spack-completion.bash` with these new flags.

Reply:
Added. I hope I did it right :p